docker/Arm64CrossCompile.Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM ubuntu:22.04 as builder
+
+ENV LANG en_US.utf8
+WORKDIR /greptimedb
+
+# Install dependencies.
+RUN apt-get update && apt-get install -y \
+    libssl-dev \
+    protobuf-compiler \
+    curl \
+    build-essential \
+    pkg-config
+
+# Install Rust.
+SHELL ["/bin/bash", "-c"]
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --no-modify-path --default-toolchain none -y
+ENV PATH /root/.cargo/bin/:$PATH
+
+
+# Install cross platform toolchain
+RUN apt-get -y update && \
+    apt-get -y install libssl-dev pkg-config g++-aarch64-linux-gnu gcc-aarch64-linux-gnu && \
+    apt-get install binutils-aarch64-linux-gnu
+
+
+
+# Download and compile Python3.10.10 with shared library support(--enable-shared).
+RUN apt-get -y install wget
```

Comment:
suggestion: we could install `wget` in the last step and make all the installation together.